### PR TITLE
feat: unittest on more ray versions.

### PR DESCRIPTION
**Pull request by zhouaihui**
TODO: client mode does not work on ray>2.4

### CODE REVIEW

**Comment by jovany-wang** [comment]
This should also close https://github.com/ray-project/rayfed/issues/195

**Comment by zhouaihui** [comment]
Further work: #199

**Review by jovany-wang — APPROVED** [review]

File: requirements.txt [hunk 1]
```diff
@@ -3,3 +3,4 @@ ray>=1.13.0
 cloudpickle
 pickle5==0.0.11; python_version < '3.8'
 protobuf
+grpcio
```
[the remark this answers is not in the document]
Comment (zhouaihui) [reply requirements.txt:6]: Any idea？Use ray 2.4 `grpcio<=1.51.3,>=1.32.0`?

[the remark this answers is not in the document]
Comment (jovany-wang) [reply requirements.txt:6]: That's okay.

**Review by jovany-wang — APPROVED** [review]

File: requirements.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (jovany-wang) [reply requirements.txt:6]: @zhouaihui I'm merging this PR. Please feel free to create a new PR to pin the grpc version.

**Pull request merged**